aspnetcore/fundamentals/aot/includes/native-aot-tutorial8.md

```diff
@@ -0,0 +1,161 @@
+:::moniker range="= aspnetcore-8.0"
+
+[!INCLUDE[](~/includes/not-latest-version.md)]
+
+ASP.NET Core 8.0 introduces support for [.NET native ahead-of-time (AOT)](/dotnet/core/deploying/native-aot/).
+
+> [!NOTE]
+> * In .NET 8, not all ASP.NET Core features are compatible with Native AOT.
+> * Tabs are provided for the [.NET CLI](/dotnet/core/tools/) and [Visual Studio](https://visualstudio.microsoft.com/downloads/) instructions:
+>   * Visual Studio is a prerequisite even if the CLI tab is selected.
+>   * The CLI must be used to publish even if the Visual Studio tab is selected.
```

Comment:
This file is a copy of the original .NET 8 version. I removed AOT is in preview in the NOTE. Does the rest apply for .NET 8?